pyvista/plotting/plotting.py

```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
 
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
+
+        >>> pl = pv.Plotter()
+        >>> pl.add_mesh(mesh, show_edges=True, color="white")
+        >>> pl.add_points(mesh.points, color="red", point_size=10)
+        >>> print(pl.camera_position)
```

Comment:
We need to add what is returned by docstring.
```suggestion
        >>> print(pl.camera_position)
        [(0.02430187165737152, 0.033604808151721954, 0.9445935746397425),
        (0.02430187165737152, 0.033604808151721954, -0.022251706570386887),
        (0.0, 1.0, 0.0)]
```